kalite/distributed/custom_context_processors.py

```diff
@@ -11,6 +11,7 @@
 
 
 def custom(request):
+    is_config_package_nalanda = "Nalanda" in settings.CONFIG_PACKAGE
```

Comment:
Can you just squash this line and https://github.com/learningequality/ka-lite/pull/2853/files#diff-aff2a0c62d4e7b4c30d8bdb171ceedb7R27?